padl/transforms.py

```diff
@@ -146,6 +151,8 @@ def __init__(self, call_info: Optional[inspector.CallInfo] = None,
         self._pd_varname = {}
         self._pd_name = pd_name
         self._pd_device = 'cpu'
+        self._pd_traceback = traceback.extract_stack()
+        self._pd_stages = None
```

Comment:
I don't think `self._pd_stages` is used anymore. It may be a left over after merging in main. Please remove